docs/docs/using-pants/advanced-target-selection.mdx

```diff
@@ -185,3 +185,58 @@ For other goals, you can leverage shell piping to partition the input targets in
 ```bash
 pants list :: | awk 'NR % 5 == 0' | xargs pants package
 ```
+
+## Referring to a group of targets
+
+If setting tags on individual targets is not feasible, there are a few other options available to refer to multiple targets.
+
+If you have an operation that you perform often on a certain group of targets, you can use the
+[cli](https://www.pantsbuild.org/2.19/reference/subsystems/cli) subsystem options to create shortcuts. For instance,
+this alias
+
+```toml title="pants.toml"
+publish-libraries = "--filter-target-type=python_distribution --filter-address-regex=\"['^src/libA/,^src/libB/']\" publish src::"
+```
+
+would you let run `pants publish-libraries` to publish all Python distributions declared in the `src/libA` and `src/libB`
+directories.
+
+If you want to be able to run an arbitrary Pants goal on a group of targets, you can create a generic target with no
+specific type named [`target`](https://www.pantsbuild.org/2.19/reference/targets/target):
+
+```python title="BUILD"
+target(name="python-libs", dependencies=["src/python/libraries/libA", "src/python/libraries/libB"])
+```
+
+If declared in the root of your workspace, you can now address the Python libraries by `//:python-libs`:
+
+```bash
+$ pants publish //:python-libs
+````
```

Comment:
Hm, using `target` for this doesn't seem to work for me, for goals like `test` or `lint`. For instance, if I have a BUILD file like:

```python
python_tests(name="tests")

target(name="tests-alias", dependencies=[":tests"])
```

Running `pants lint path/to:tests` runs black and isort etc. while `pants lint path/to:tests-alias` does nothing. Similarly, running `pants test path/to:tests-alias` gives a hard error:

```
08:25:43.00 [WARN] No applicable files or targets matched. The `test` goal works with these target types:

  * experimental_test_shell_command
  * python_test
  * shunit2_test

However, you only specified target arguments with these target types:

  * target

Please specify relevant file and/or target arguments. Run `pants --filter-target-type=experimental_test_shell_command,python_test,shunit2_test list ::` to find all applicable targets in your project, or run `pants --filter-target-type=experimental_test_shell_command,python_test,shunit2_test filedeps ::` to find all applicable files.
```

`publish` seems to be behave similarly.

So, I think using `target` to create a group for other targets to use as _dependencies_ is fine, but it doesn't work for running goals.

Reply:
Oh yes, I've just realized that, thanks! I've made the changes.